src/supabase/queries/cases.ts

```diff
@@ -71,6 +71,34 @@ export async function getCasesByIds(caseIds: CaseUid[]): Promise<Case[]> {
   }
 }
 
+export async function uploadCase(caseId: CaseUid | undefined) {
```

Comment:
Sorry might've missed this but did you write these queries (`uploadCase` and `containsDuplicateCase`)?

Reply:
I believe so yes, but they would've been in a previous I'm pretty certain